src/templates/upgrade.tsx

```diff
@@ -358,49 +411,57 @@ const UpgradePage = ({
           <DesktopBreadcrumbs slug={slug} startDepth={1} />
           <MobileBreadcrumbs slug={slug} startDepth={1} />
```

Comment:
Why we are using 2 separate Breadcrumbs having the style?  Can anybody explain to me why we are doing this?